wandb/sdk/wandb_run.py

```diff
@@ -3123,38 +3123,81 @@ def log_model(
                     - `/local/directory`
                     - `/local/directory/file.txt`
                     - `s3://bucket/path`
-            model_name: (str, optional) An artifact name. String containing only the following alphanumeric characters: dashes, underscores, and dots.
+            name: (str, optional) An artifact name. String containing only the following alphanumeric characters: dashes, underscores, and dots.
```

Comment:
```suggestion
            name: (str, optional) A name to assign to the model artifact that the file contents will be added to. String containing only the following alphanumeric characters: dashes, underscores, and dots. This will default to the basename of the path prepended with the current run id  if not specified.
```

Reply:
Please try not to make changes to the text in red. That shows the current status and if you make changes, it makes the pr comments harder to read.